packages/cli/src/util/link/setup-and-link.ts

```diff
@@ -188,7 +187,7 @@ export default async function setupAndLink(
         [sourcePath],
         createArgs,
         org,
-        !isFile,
+        true,
```

Comment:
Should this param be removed also?

Reply:
No, it's still used dynamically in the `deploy` command.